### ISSUE TITLE
The manual claims it will not send passwords by default, this is not true

### ISSUE THREAD

**Issue by aaronjensen** [appsignal/appsignal-elixir#87]
From the [Getting started](https://hexdocs.pm/appsignal/gettingstarted.html#using-phoenix-s-filter_parameters-configuration) page:

"If :filter_parameters is not set, Phoenix will default to ["password"]. This means that a Phoenix app will not send any passwords to AppSignal without any configuration."

However, my first error on the sign in page sent the user's password to appsignal 😞 

This seems to work:

```elixir
config :appsignal, :config,
  filter_parameters: ["password"]
```

**Comment by tombruijn** [comment]
Thanks @aaronjensen for the report!
We'll have a look to see what's going wrong there

**Comment by arjan** [comment]
Can you verify that this still occurs with the latest release? (0.11.2) because we merged an issue related to this, #73. But maybe Phoenix does not use a default filter parameters list.

**Comment by aaronjensen** [comment]
Ah, indeed. I was on an older version before. It appears to be fixed in 0.11.2. Thanks.
